### PR TITLE
HyperShift: Enable IPV6 dual stack

### PR DESCRIPTION
**Pull request by LiangquanLi930**

### CODE REVIEW

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-agent-ovn-mce-dualstack-conformance

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-agent-ovn-mce-dualstack-conformance

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-agent-ovn-mce-dualstack-conformance

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-agent-ovn-mce-dualstack-conformance

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-agent-ovn-mce-dualstack-conformance

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-agent-ovn-mce-dualstack-conformance

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-agent-ovn-mce-dualstack-conformance

File: ci-operator/step-registry/hypershift/mce/agent/dualstack/conformance/hypershift-mce-agent-dualstack-conformance-workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    - ref: wait
+    - chain: hypershift-mce-agent-create
+    env:
+      LVM_OPERATOR_SUB_SOURCE: qe-app-registry
+      PACKET_PLAN: "m3.large.x86"
+      USER_DATA_FILENAME: "user-data"
+      NUM_EXTRA_WORKERS: "3"
+      IP_STACK: v4v6
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v4v6
```
Comment (aleskandro) [inline ci-operator/step-registry/hypershift/mce/agent/dualstack/conformance/hypershift-mce-agent-dualstack-conformance-workflow.yaml:30]:
```suggestion
        IP_STACK=v4v6
        HOST_IP_STACK=v4v6
```

**Comment by aleskandro** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-agent-ovn-mce-dualstack-conformance

**Pull request closed without merging**